docs/rest-api-getting-started.md

```diff
@@ -0,0 +1,391 @@
+    "vip_dhcp_allocation": true
+}
+```
+
+## Modify and Generate Discovery ISO
+
+### Get Public Key
+* In case it is locally stored, you will usually find it at `~/.ssh/id_rsa.pub`
+
+### Patch Cluster
+* `PATCH /v2/clusters/{cluster_id}`
+* operationId: `V2UpdateCluster`
+
+
+* Note:
+  - Configure your public SSH key here to gain SSH access to your live cluster (post-install).
+  - Some more advanced use-cases, which are out of scope for this document, may include multiple Infra Envs linked to the same cluster. In such a case, the SSH key you set here will be the key you use to access your cluster post-installation.
+
+
+
+```bash
+curl -X PATCH -H "Content-Type: application/json" \
+    -d '{"http_proxy":"","https_proxy":"","no_proxy":"","pull_secret":"","ssh_public_key":"<public_key_here>"}' \
+    <HOST>:<PORT>/api/assisted-install/v2/clusters/<cluster_id>
+```
+
+### Patch InfraEnv
+* `PATCH /v2/infra-envs/{infra_env_id}`
+* operationId: `UpdateInfraEnv`
+
+
+* Note: Configure your public SSH key here to gain SSH access to your hosts in the discovery phase (pre-install).
+
+```bash
+curl -X POST -H "Content-Type: application/json" \
+    -d '{"proxy":{"http_proxy":"","https_proxy":"","no_proxy":""},"ssh_authorized_key":"<public_key_here>","pull_secret":"","image_type":"full-iso"}' \
+    <HOST>:<PORT>/api/assisted-install/v2/infra-envs/<infra_en_id>
+```
+
+## Inspect InfraEnv Status
+* `GET /v2/infra-envs`
+* operationId: `ListInfraEnvs`
+```bash
+curl <HOST>:<PORT>/api/assisted-install/v2/infra-envs/<infra_env_id> | python -m json.tool
+```
+### Result
+```json
+{
+  "cluster_id": "948b8ba9-798f-4310-bb08-67a6b56ebdcc",
+  "cpu_architecture": "x86_64",
+  "created_at": "2021-12-16T14:56:30.805445Z",
+  "download_url": "http://<HOST>:<PORT>/images/4f630ba3-0297-4d99-a7c7-f62e7bca1332?arch=x86_64&type=full-iso&version=4.8",
+  "email_domain": "Unknown",
+  "expires_at": "0001-01-01T00:00:00.000Z",
+  "href": "/api/assisted-install/v2/infra-envs/4f630ba3-0297-4d99-a7c7-f62e7bca1332",
+  "id": "4f630ba3-0297-4d99-a7c7-f62e7bca1332",
+  "kind": "InfraEnv",
+  "name": "testcluster_infra-env",
+  "openshift_version": "4.8",
+  "proxy": {
+    "http_proxy": "",
+    "https_proxy": "",
+    "no_proxy": ""
+  },
+  "pull_secret_set": true,
+  "ssh_authorized_key": "<public ssh key here>",
+  "type": "full-iso",
+  "updated_at": "2021-12-16T17:51:40.180877Z",
+  "user_name": "admin"
+}
+```
+
+## Get InfraEnv Image Download URL
+* `GET /v2/infra-envs/{infra_env_id}/downloads/image-url`
+* operationId: `GetInfraEnvDownloadURL`
+
+```bash
+curl <HOST>:<PORT>/api/assisted-install/v2/infra-envs/<infra_env_id>/downloads/image-url | python -m json.tool
```

Comment:
don't people more familiar with jq? 
just noticed it in this call, but it's relevant for all the doc.

Reply:
Can you please elaborate?
Do you mean like it is used in [restful-api-guide.md](https://github.com/openshift/assisted-service/blob/master/docs/user-guide/restful-api-guide.md )?

Reply:
I mean instead of using `python -m json.tool` use `jq` https://www.baeldung.com/linux/jq-command-json

Reply:
tested with `jq '.'` and works as expected. Done.